addons/project/static/src/js/widgets/task_kanban_state_switch_widget.js

```diff
@@ -0,0 +1,54 @@
+/** @odoo-module **/
+
+import core from 'web.core';
+import fieldRegistry from 'web.field_registry';
+
+import { StateSelectionWidget } from 'web.basic_fields';
+import { sprintf } from "@web/core/utils/strings";
+
+export const KanbanStateSwitchWidget = StateSelectionWidget.extend({
+
+    on_attach_callback() {
```

Comment:
The blank line is not needed. 😇 
```suggestion
    on_attach_callback() {
```